public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
```

Comment:
You are right! Thank you!

I will also try to set up a continuos integration service in a separate pull request.